Docker/install.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Configure environment
+export DEBIAN_FRONTEND="noninteractive"
+
+# Configure user squeezeboxserver
+useradd squeezeboxserver
+usermod -u 99 squeezeboxserver
+usermod -g 100 squeezeboxserver
+usermod -d /home squeezeboxserver
+chown -R squeezeboxserver:users /home
+
+# Allow acces to /dev/snd
+usermod -a -G audio squeezeboxserver
```

Comment:
Could we move this to the Dockerfile? There is not that much happening here anymore. Of course, we want to keep the number of layers low so it then should go into one RUN command.

Reply:
Agree, no need for this file anymore.

Reply:
Thought so :-)